pkg/cli/cli.go

```diff
@@ -307,7 +308,11 @@ func actionStart(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	defer bodyReader.Close()
+	defer func() {
+		if err := bodyReader.Close(); err != nil {
+			log.Err(err).Msg("failed to close bodyReader")
```

Comment:
maybe it's better to specify the log level explicitly.

Reply:
I forgot to specify the log level. I will set it to Error.